.github/workflows/update-foundry-prove.yml

```
@@ -0,0 +1,27 @@
+name: 'Update Foundry prove'
+on: push
+jobs:
```

```suggestion
name: 'Update Foundry prove'
on: workflow_dispatch
jobs:
```

This will make it so that it only runs when the user says to run it.

---

Thank you! I will make it `on: workflow_dispatch` before that the changes make it on master, the `on: push` is for debugging purposes for now.